test/MatrixSets/MatrixZonotope.jl

```diff
@@ -1,4 +1,4 @@
-using LazySets, Test
+using LazySets, Test, IntervalMatrices
```

Comment:
I would prefer to add the `@static if ...` guard below to make the tests executable without adding packages to your environment.